package sec02.exceptions

fun erroneousCode() {
    // 예외를 보려면 다음 줄의 주석을 해제
     val i = "i$".toInt()
}

fun main() {
    erroneousCode()
}